tests/validation/tests/v3_api/common.py

```diff
@@ -2525,3 +2530,55 @@ def create_pv(client, nfs_ip):
     assert capacitydict['storage'] == '50Gi'
     assert pv_object['type'] == 'persistentVolume'
     return pv_object
+
+
+def delete_resource_in_AWS_by_prefix(resource_prefix):
+    """
+    :param resource_prefix: the prefix of resource name
+    :return: None
+    """
+    # delete nodes
```

Comment:
Can we add comment that nodes of the local cluster are also deleted here ?

Reply:
comment is added